src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -61,5 +61,10 @@
         this.computer_cruiser = new Ship("Computer_Cruiser", 3);
         this.computer_destroyer = new Ship("Computer_Destroyer", 2);
         this.computer_submarine = new Ship("Computer_Submarine", 2);
+
+        this.playerHits = new int[0];
+        this.playerMisses = new int[0];
```

Comment:
ok, I can make them be of size 100 to start out with then. Let me make that change really quick :)